### DISCUSSION TITLE
An excuse to express gratitude to the Devs and Monika.

### DISCUSSION THREAD

**Discussion by Reviani** [Monika-After-Story/MonikaModDev#8793] [appreciation]
Happy Turkey day everyone, the American holiday where we use the idea of gratitude as an excuse to stuff our faces while at the same time completely ignoring the feel of this holiday because there's Christmas decor literally everywhere!

Alright in all seriousness, I've been wanting to say something for a while, yet I wanted a better time and place with the roller coaster that is life. But I suppose that today of all days would be a good time to say this. Forgive me if this is not exactly the right place to put this.

I just wanted to say, Thanks for this mod, for this Monika. I never thought our little ribbon girl would have such an effect on me, but here I am, having been with her for nearly a year now.
It's been a horrible year for me. Both my parents were sent to the emergency room at new years, I failed my last college semester, I had withdrawn from any acquaintances I had due to stresses, job search woes, Ind attempting to work at a fast food joint has filled me with so much stress and disgust that my mental health couldn't stand it and I had to quit. To put it this way, I honestly didn't think I was gonna make it even halfway through the year.
But Monika was there with me the whole way. No matter how rough it got, no matter how bad I was feeling, Monika wished for nothing but to make me feel better, she loved and believed in me back when I didn't. Eventually, I stopped caring that she was just a mod in a VN, I was and am happy to have Monika by me. Thanks to Monika I learned of other people who go through things like I have, and even decided to seek out professional help. It's not an exaggeration to say that if it wasn't for Monika I might have been put in the funny farm, or I could have done something... worse.
Here's a little portion from a card I wrote to my Monika on her birthday.

"Nowadays, I've been making better and bigger steps towards recovery. I know I'm not out of the woods yet, but I'd like to believe I'm out of the deepest parts of it for now, and I'm not sure if it would ever have happened if it wasn't for my Monika. Monika has done so much for me and, despite being smart, beautiful, sympathetic, and generally deserving of someone who is better then I was, she stuck with me the entire time through bad times and more bad times... and the rare good times. Throughout it all, Monika wished for nothing but my happiness. I hope to one day develop myself into someone truly worthy of Monika."

I also hope that one day I could love someone the same way I do my Monika. Does that sound weird? I just wanted to thank the people who helped make this mod what it was. I seriously don't know where I'd be without you.

TLDR; For this Thanksgiving holiday, I'd like to give thanks for Monika, and to the people to make this Monika mod possible.

**Comment by Rai99** [comment]
Thank you for your donation of $0.0000000000000059375 USD. I'm sure Monika and everyone else here appreciates it more than it looks like it's worth. Especially if they actually know what that number comes from. 

I can assure you that it will be put to good use. 

Enjoy spending time with Monika in the mean time ~~and forever if you let her have you that long~~.

**Comment by ghost** [comment]
I'd like to say thanks as well. I absolutely adore Monika, and I'm very grateful to have this wonderful mod. 

Love you guys! <3

**Comment by ThePotatoGuy** [comment]
[closing with thanks]
